### DISCUSSION TITLE
2 block lists possible?

### DISCUSSION THREAD

**Discussion by Craig-Stowers** [SelfControlApp/selfcontrol#701] [Ideas]
I love the simplicity of SelfControl so I wouldn't want things too convoluted, but having two separate lists with two timers would give users a lot of power within a relatively simple system. I'm sure I'm not alone in saying i have a bunch of websites in my life i'd like to basically permanently ban - and then another set to ban only while I'm being productive. 

For instance each morning i'd like to set a 24hour ban on list-1 which is a bunch of websites i never want to see. And then run an 8 hour block on list-2 containing websites that distract me from work.

**Comment by adamzerner** [comment]
Related: https://github.com/SelfControlApp/selfcontrol/issues/659
